be/src/olap/rowset/segment_v2/inverted_index_file_writer.cpp

```diff
@@ -120,6 +122,56 @@ int64_t InvertedIndexFileWriter::headerLength() {
     return header_size;
 }
 
+Status InvertedIndexFileWriter::add_into_searcher_cache() {
```

Comment:
Can you call search get function used in query to fill the cache automatically?

Reply:
No